packages/jupyter-ai/jupyter_ai/models.py

```diff
@@ -263,3 +263,45 @@ class ListSlashCommandsEntry(BaseModel):
 
 class ListSlashCommandsResponse(BaseModel):
     slash_commands: List[ListSlashCommandsEntry] = []
+
+
+class ListOptionsEntry(BaseModel):
+    id: str
+    # includes the command prefix. e.g. "/clear", "@file".
+    label: str
+    description: str
+    only_start: bool
+    # only allows autocomplete to be triggered if command is at start of input
```

Comment:
Can you add docstrings for each of these attributes that describe exactly what each attribute is supposed to contain, preferably with examples? Some initial questions I had when reading through this model as a developer:

- What is the difference between `id` and `label`? 
- What does `only_start` do, and why would it be set to `True`?

Reply:
- `label` is the actual completion text.
  - For non-arg commands it would be id + space (" ").
  - For arg commands it would be "{id}:{arg} " or not end with a space if it is a partially complete arg like for directories in @file.
  - this is similarly defined in your original implementation. I just added the "/" or "@" to the id to help generalise it.
  - Should I rename it to `text` or something else less confusing?
- `only_start` is `True` for slash commands
  -  is mainly to differentiate between slash commands that can only be triggered at the beginning of the prompt vs @ commands that can be triggered anywhere.